suite-common/suite-types/src/modal.ts

```diff
@@ -171,4 +171,8 @@ export type UserContextPayload =
       }
     | {
           type: 'stake';
+      }
+    | {
+          type: 'confirm-stake-eth';
```

Comment:
This modal should be over the staking form. 

Just create modal locally as it is for example in `FirmwareInstallation` component - `ReconnectDevicePrompt`

Reply:
https://github.com/everstake/trezor-suite/pull/4/commits/5600ebf2fc8bc6f068736fce383ee6bb9e40bee0

btw, I think now I can remove the drafts logic from `useStakeEthForm` as you wanted in the previous PR, since opening the "Confirm" modal doesn't close the modal with the form anymore.